### ISSUE TITLE
Auto update to proc-macro2 breaks proc-quote

### ISSUE THREAD

**Issue by EverlastingBugstopper** [Goncalerta/proc-quote#15]
proc-macro2 introduced some breaking changes in their latest release. this crate recently automatically updated all dependencies and it seems to be incompatible w/this crate 😄 

https://github.com/Goncalerta/proc-quote/commit/d262bd99922f2bb5eb66fece5a7198abe5425f3a#diff-80398c5faae3c069e4e6aa2ed11b28c0R20

example failed install log
```bash
Compiling liquid-derive v0.19.0
error[E0053]: method `to_tokens` has an incompatible type for trait
  --> /Users/me/.cargo/registry/src/github.com-xxxxxxxx/liquid-derive-0.19.0/src/filter/display.rs:34:5
   |
34 |     fn to_tokens(&self, tokens: &mut TokenStream) {
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `proc_quote::__rt::TokenStream`, found struct `proc_macro2::TokenStream`
   |
   = note: expected type `fn(&filter::display::Parameters<'a>, &mut proc_quote::__rt::TokenStream)`
              found type `fn(&filter::display::Parameters<'a>, &mut proc_macro2::TokenStream)`
note: Perhaps two different versions of crate `proc_macro2` are being used?
  --> /Users/me/.cargo/registry/src/github.com-xxxxxxxxxx/liquid-derive-0.19.0/src/filter/display.rs:34:5
   |
34 |     fn to_tokens(&self, tokens: &mut TokenStream) {
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0053]: method `to_tokens` has an incompatible type for trait
   --> /Users/me/.cargo/registry/src/github.com-xxxxxxx/liquid-derive-0.19.0/src/filter_parameters.rs:300:5
    |
300 |     fn to_tokens(&self, tokens: &mut TokenStream) {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `proc_quote::__rt::TokenStream`, found struct `proc_macro2::TokenStream`
    |
    = note: expected type `fn(&filter_parameters::FilterParameter<'a>, &mut proc_quote::__rt::TokenStream)`
               found type `fn(&filter_parameters::FilterParameter<'a>, &mut proc_macro2::TokenStream)`
note: Perhaps two different versions of crate `proc_macro2` are being used?
   --> /Users/me/.cargo/registry/src/github.com-xxxxxxx/liquid-derive-0.19.0/src/filter_parameters.rs:300:5
    |
300 |     fn to_tokens(&self, tokens: &mut TokenStream) {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors
For more information about this error, try `rustc --explain E0053`.
error: Could not compile `liquid-derive`.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `cargo-generate v0.4.0`, intermediate artifacts can be found at `/var/folders/xx/xxxxxxxxxxx/T/cargo-installaxxxxx`
```

**Comment by Goncalerta** [comment]
Sorry! 😅
I was in a hurry this morning and didn't realise I was releasing the wrong version. Worse even, it appears I had skipped `0.2.3` completely.

I've just yanked `0.2.4` and re-released it as `0.3.0`. `liquid-derive` now compiles on my machine so I guess this is fixed.

**Comment by EverlastingBugstopper** [comment]
No worries! Happens to all of us 😄
